examples/source/user-consent/Geolocation-based_Consent_Flow.html

```diff
@@ -75,51 +75,98 @@
 </head>
 <body>
   <!-- ## Setting up amp-geo -->
-  <!-- First we need to setup the [`amp-geo`](/content/amp-dev/documentation/components/reference/amp-geo-v0.1.md) extension. We'll use the present `eea` which includes all countries of the European Economic Area. -->
+  <!-- First we need to setup the [`amp-geo`](/content/amp-dev/documentation/components/reference/amp-geo-v0.1.md) extension. We'll use the iso code `us` which represents the United States, `preset-eea`, and [`unknown`](https://amp.dev/documentation/components/amp-geo/?format=websites#operation)-->
   <amp-geo layout="nodisplay">
     <script type="application/json">
       {
         "ISOCountryGroups": {
-          "eu": [ "preset-eea" ]
+          "eea": [ "preset-eea", "unknown" ],
+          "us": [ "us" ]
         }
       }
     </script>
   </amp-geo>
 
+  
   <!-- ## Defining the consent flow -->
   <!--
-    We define a basic consent flow, similar to [this sample](/content/amp-dev/documentation/examples/documentation/Basic_User_Consent_Flow.html). he flow should only trigger for users in the EU, so we set the flag: `"promptIfUnknownForGeoGroup": "eu"`.
+    The flow should rely on amp-geo to determine which config to show based upon the user's location, so we set the flag: [`geoOverride`](/content/amp-dev/documentation/components/reference/amp-consent.md).
```

Comment:
nit: link to https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/amp-consent.md#geooverride instead